docs/styleguide.config.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path')
+
+module.exports = {
+  components: path.resolve(__dirname, '../react/{Button,Toggle}/*.{js,jsx}'),
```

Comment:
You'll have to understand that this configuration entry is going to change as soon as somebody write `README.md` files for the other components.

Reply:
I think we could just put '../react/**/*.jsx' and [`skipComponentsWithoutExample`](https://react-styleguidist.js.org/docs/configuration.html#skipcomponentswithoutexample)

Reply:
We may, but I did not seek to far. As said, we should adapt such a configuration with the other component's integration. The main part is here.